### PR TITLE
DOC : Added environment variable details to nsls2 docs

### PR DESCRIPTION
**Pull request by jrmlhermitte**
Before I forget, this is an important useful step. Could be nice to keep this somewhere in the docs.

Please don't merge yet. I want to test it over again.

### CODE REVIEW

**Comment by jrmlhermitte** [comment]
@yugangzhang and @mrakitin might want to give it a try (if i missed something. building now....)
thanks @yugangzhang for the details!

File: source/conda.rst [hunk 1]
```diff
@@ -169,6 +169,43 @@ JupyterHub.
...
+
+        HDF5_PLUGIN_PATH=/path/to/HDF5Plugin
+
+   This is added by simply adding this line above to the "argv" field:
+
+    .. code-block:: json
+
+        {
+         "argv": [
+         "HDF5_PLUGIN_PATH=/path/to/HDF5Plugin",
```
Comment (tacaswell) [inline source/conda.rst:190]: This should ideally be a path into the enviroment!

Comment (jrmlhermitte) [reply source/conda.rst:190]: so `/home/dallan/conda_envs/myenv/path/to/HDF5Plugin` then?
Also, added a note to make it clear that this should precede the executable in args. this is like running
`ENV1=123 ENV2=234 myexec`

**Comment by mrakitin** [comment]
Probably unrelated error to the changes in this PR:
```python
Warning, treated as error:
/home/travis/build/NSLS-II/docs/source/_cookbook/count_with_table.py failed to execute correctly: Traceback (most recent call last):
  File "/home/travis/build/NSLS-II/docs/source/_cookbook/count_with_table.py", line 23, in <module>
    from bluesky.examples import det  # a simulated detector
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/bluesky/examples.py", line 10, in <module>
    moved to ophyd.sim. A version of ophyd v0.8.0 or greater is required.""")
ImportError: 
The simulated hardware objects in the bluesky.examples module have been
moved to ophyd.sim. A version of ophyd v0.8.0 or greater is required.
make: *** [html] Error 1
```

**Comment by jrmlhermitte** [comment]
could be good to wait on [this](https://github.com/NSLS-II/docs/pull/72) then.

**Pull request closed without merging**